packages/frontend/public/index.html

```diff
@@ -12,6 +12,7 @@
         color: #111;
       }
     </style>
+    <script type="text/javascript" src="./Jdenticon.min.js"></script>
```

Comment:
Can we find a react component library that we can pull in as a dependency in package.json in the FE?

Reply:
Unfortunately not, I had searched, but there was no working react library at the time when I started implementing this.